InventoryDistanceBasedSourceSelection/Model/DistanceProvider/GoogleMap/GetLatLngFromAddress.php

```diff
@@ -41,16 +42,22 @@ class GetLatLngFromAddress implements GetLatLngFromAddressInterface
     private $latLngInterfaceFactory;
 
     /**
+     * @deprecated
+     *
      * @var Json
      */
     private $json;
 
     /**
+     * @deprecated
+     *
      * @var GetApiKey
      */
     private $getApiKey;
 
     /**
+     * @deprecated
+     *
      * @var AddressToComponentsString
      */
     private $addressToComponentsString;
```

Comment:
You may remove private properties and their assignment in the constructor since they are not used anymore.

Reply:
done